packages/ui5-middleware-iasync/README.md

```diff
@@ -0,0 +1,75 @@
+# UI5 middleware for syncing interactions across browsers
+
+Middleware for [ui5-server](https://github.com/SAP/ui5-server), syncing interactions between different browsers, hopefully somewhat easing manual testing :)  
+
+## Install
+
+```bash
+npm install ui5-middleware-iasync --save-dev
+```
+
+## Configuration options (in `$yourapp/ui5.yaml`)
+
+- `https: <boolean>` default: false  
+whether to use the middleware via SSL/wss
+- `httpModule: <string>`, default: undefined  
+capability to e.g. use `http2`
+- `port: <integer>`, default: 3000  
```

Comment:
shouldn't we change the default? Does this collide with any other port?

Reply:
browsersync default is 3000 - currently no conflict

Reply:
Ok